src/Gzero/Core/Parsers/ArrayParser.php

```diff
@@ -93,20 +93,24 @@ public function parse(Request $request)
     {
         if ($request->has($this->name)) {
             $this->applied = true;
+            $this->operation = 'in';
```

Comment:
@bartero @michalstrzelecki I disagree with this one, from my point of view, `parse` method is about setting a new value, not default one and it should care only about the fact that new value is a valid one before setting it.

The default value should be set when `protected $operation` is defined.

Reply:
@bartero @michalstrzelecki Now the default operation will be `null` until you run `parse` method.  therefore having getOperation() and setOperation() is pointless.

`$parser = new ArrayParser()`
`$parser->setOperation('not in')`
`$parse->parse($input) - it will require unnecessary check for default value`